lib/bump/dependency_metadata_finders/base.rb

```diff
@@ -70,9 +70,8 @@ def look_up_changelog_url
       def look_up_release_url
         @release_url_lookup_attempted = true
 
-        release = github_client.releases(github_repo).find do |r|
-          r.name.to_s.gsub(TAG_PREFIX, "") == dependency.version
-        end
+        release = github_client.releases(github_repo).
+                  find { |r| r.name.to_s.end_with?(dependency.version) }
```

Comment:
I'm guessing this would mean that `1.0.0` would also match `21.0.0`, right? Could fix by changing to a regex along the lines of `[^0-9]#{Regexp.escape(dependency.version)}\z`.